packages/@vuepress/bundler-webpack/src/dev/createDevServerConfig.ts

```diff
@@ -28,11 +28,11 @@ export const createDevServerConfig = (
     },
     host: app.options.host,
     hot: true,
-    onAfterSetupMiddleware: (expressApp, server) => {
+    onAfterSetupMiddleware: ({ app: expressApp }, server) => {
```

Comment:
Then the `server` param could be removed now. And the types should be updated.

Reply:
I will fix that tomorrow morning